recipes/elk/webview.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (Ferdium) => {
+    Ferdium.loop(() => {
+        // FIXME: Don't assume 0th element exists
+        const notifications = document.querySelectorAll("[href$=notifications]")[0].outerText;
+        let parsedValue = Ferdium.safeParseInt(notifications);
+
+        Ferdium.setBadge(parsedValue);
+    });
```

Comment:
There should be some lines that are used to load `service.css`. please refer to the template and add them.

Reply:
Is it necessary? I don't have a service.css file

Reply:
That option is for the user to be able to override if they choose to - not for the recipe to have a service.css